### DISCUSSION TITLE
Problem installing

### DISCUSSION THREAD

**Discussion by Jakezys** [Nitrux/nitrux-bug-tracker#148] [Q&A] [answered]
When I try to install Nitrux, it gives me an error saying that it con not access the selected time zone path.

**Comment by UriHerrera** [comment, marked as answer]
_**This is not a problem of the distribution**_. The storage device you're using (whatever it is, you don't provide any information) does not have enough storage to hold the root partition.

See  [Install fails with error: cannot access selected timezone path. Wrong/Invalid path: /tmp/calamares-root-[$temp-dir]/usr/share/zoneinfo/[$TIMEZONE] #100](https://github.com/Nitrux/nitrux-bug-tracker/issues/100).

---

I'd like you to please read the [tutorial to install the distribution](https://nxos.org/tutorial/how-to-install-nitrux/) for system requirements information, including virtualization instructions.

**Reply by Jakezys** [reply]
ok thanks
